app/src/main/res/layout/bottom_left_overlay.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="listener"
+      type="org.oppia.android.app.spotlight.SpotlightNavigationListener" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/overlayConstraintLayout"
```

Comment:
For this & all other IDs in the overlay layouts, please make sure:
- They use consistent snake_case naming.
- They are named with contextually clear names ('arrow' is extremely generic and vague--remember that these IDs are global).

Reply:
done